### PR TITLE
task #3509 navbar made transparent in /login /signup /reset-password

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

File: hosting/templates/hosting/includes/_navbar_transparent.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        </li>
+                        <li>
+                            <a href="{% url 'hosting:notifications' %}">
+                                <i class="fa fa-bell"></i> {% trans "Notifications "%}
+                            </a>
+                        </li>
+                        <li><a href="{% url 'hosting:logout' %}"><i class="glyphicon glyphicon-lock"></i>{% trans "Logout"%} </a></li>
+                      </ul>
+                    </li>
+                    <!--
```
Comment (pcoder) [inline hosting/templates/hosting/includes/_navbar_transparent.html]: I guess, we can remove the commented code ?

**Comment by verysanghee** [comment]
tested and looks good!

File: hosting/templates/hosting/base_short.html [hunk 2]
```diff
@@ -48,78 +48,80 @@
...
+                                </li>
+                                <li>
+                                    <a href="{% url 'hosting:notifications' %}">
+                                        <i class="fa fa-bell"></i> {% trans "Notifications "%}
+                                    </a>
+                                </li>
+                                <li><a href="{% url 'hosting:logout' %}"><i class="glyphicon glyphicon-lock"></i>{% trans "Logout"%} </a></li>
+                              </ul>
+                            </li>
+                            <!--
```
Comment (pcoder) [inline hosting/templates/hosting/base_short.html]: Could we have the commented code removed?

**Comment by pcoder** [comment]
login-screen-transparent-navbar

**Comment by pcoder** [comment]
In the above screenshot, I see the following issues.

1. We don't have a menu in the desktop version, I wonder how we have that in the mobile version?
2. The datacenterlight logo in the navbar seems to be overlapped by the form. I think this is not a desirable behavior.
3. The footer does not look clear with the image as the background.

@verysanghee could you please confirm?

**Comment by verysanghee** [comment]
@pcoder All valid points, thanks. I found them myself while testing too and wrote tickets accordingly. 
1. I think it happens with browser size for responsiveness, and we never cleaned up this part. It appears in staging as well. I assigned a cleanup in this ticket. https://redmine.ungleich.ch/issues/3605
2. I've noticed it in smaller desktop browser as well, however it appears to be clean on mobile. 
3. I wrote a ticket for highlighting footer text color on mouseover here https://redmine.ungleich.ch/issues/3604

@pcoder I think this PR can be merged as it is and the rest of fixes can be taken cared in new PRs, what do you think? 
@tiwariav feel free to give feedback on above points discussed!

**Comment by tiwariav** [comment]
@pcoder commented code removed.

**Comment by tiwariav** [comment]
I've also seen the above issue, which occurs since the alignment is by flex, justify-content. A suitable padding/margin on the top should help.

**Comment by tiwariav** [comment]
also completed task #3605 in the same PR, since it involved changes to the same file and would have resulted in unnecessary conflicts.

**Comment by verysanghee** [comment]
tested again, looks good.

**Comment by verysanghee** [comment]
on iphone the signup page looks not good @tiwariav needs to be fixed 
img_3865

**Pull request merged**